gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -814,13 +879,8 @@ def plug_network_function_device_interfaces(self, device_data,
             raise exceptions.ComputePolicyNotSupported(
                 compute_policy=device_data['service_details']['device_type'])
 
-        token = self._get_token(device_data.get('token'))
-        if not token:
-            return None
-
-        image_name = self._get_image_name(device_data)
-        if image_name:
-            self._update_vendor_data(device_data)
+        token = device_data['token']
+        tenant_id = device_data['tenant_id']
```

Comment:
We used tenant_id and admin_tenant_id in many places. Can we use only one name commonly everywhere.